rtc/hello_rtc/hello_rtc.c

```diff
@@ -32,6 +32,9 @@ int main() {
     rtc_init();
     rtc_set_datetime(&t);
 
+    // clk_sys is >2000x faster than clk_rtc, so datetime is not transferred completely when rtc_get_datetime() is called
```

Comment:
Was the comment here supposed to say `rtc_set_datetime()` ?
And perhaps "not updated immediately" makes more sense than "not transferred completely" ?

Reply:
Issue is that the set function is asynchronous, which is not expected. So the patch here is more or less a workaround. 

I am not a native english speaker, if you think your sentence makes more sense, I can take it. From my pov there are 2 controllers, the fast MCU sends multiple bytes, but the slow RTC didn't get all of it before the getter is called. 
I think a comment for the setter which describes this behavior would be good

Reply:
Decision for @kilograham :wink: